action.yml

```diff
@@ -1,34 +1,77 @@
 name: Dispatch CI
 description: GitHub Action to trigger ci.yml workflow_dispatch event via GitHub API
 
+inputs:
+  major_type:
+    type: string
+    required: false
+    default: 'dynamic'
+  minor_type:
+    type: string
+    required: false
+    default: 'dynamic'
```

Comment:
Add note about the inputs to the README like we do for other gha-* repos.

Reply:
Done